src/filesystem/File.js

```diff
@@ -103,7 +110,7 @@ define(function (require, exports, module) {
         // for a default value; otherwise it could be the empty string, which is
         // falsey.
         if (this._contents !== null && this._stat) {
-            callback(null, this._contents, this._encoding, this._stat);
+            callback(null, this._contents, this._encoding, this._preserveBOM, this._stat);
```

Comment:
Remove the preserveBOM marker from callback. It should be consumed internally and kept in File prototype only.